File: IAL-002/Listas/4 - Funções/05.py
def divisao(p, q):
    # Recebe: um inteiro p (p>=0) e um inteiro q (q>0).
    # Retorna: um inteiro com o quociente da divisao inteira de p por q.
    # Obs: nao usar os operadores de divisao e/ou multiplicacao.
    div = 0
    while(p >= q):
        p = p-q
        div = div+1
    return div

File: ILP-010/Listas/1-Funcoes/05.c
int resto(int p, int q){
//Recebe: dois inteiros p e q (p >= 0  e q > 0).
//Retorna: o resto da divisao inteira de p por q.
//Obs: nao usar operador de resto.
  while(p>=q)
    p-=q;
  return p;
}
